online-shop/src/app/store/actions/product.actions.ts

```diff
@@ -0,0 +1,115 @@
+import {Action} from '@ngrx/store';
+import {Product} from '../../models/product';
+
+export enum EProductActions {
```

Comment:
Not sure what the E comes from :) Also, it might be an idea to split up the actions into several files (e.g. one for the detail page and one for the list page).

Reply:
From an example that I seen I noticed that it follows a coding convention similar to the C# standard('I' at the front for interface, 'E' for enum..)

Reply:
I'm not sure if it's okay and I was meaning to ask you